pyvista/plotting/axes_actor.py

```diff
@@ -189,195 +429,351 @@ def tip_length(self) -> tuple:  # numpydoc ignore=RT01
+        >>> axes_actor.label_position = 0.3
+        >>> axes_actor.label_position
         (0.3, 0.3, 0.3)
-        >>> axes.axes_actor.label_position = (0.1, 0.4, 0.2)
-        >>> axes.axes_actor.label_position
+        >>> axes_actor.label_position = (0.1, 0.4, 0.2)
+        >>> axes_actor.label_position
         (0.1, 0.4, 0.2)
 
         """
         return self.GetNormalizedLabelPosition()
 
     @label_position.setter
-    def label_position(self, length):  # numpydoc ignore=GL08
-        if isinstance(length, Iterable):
+    def label_position(self, length: Union[float, Sequence[float]]):  # numpydoc ignore=GL08
+        if isinstance(length, Sequence):
             self.SetNormalizedLabelPosition(length[0], length[1], length[2])
         else:
             self.SetNormalizedLabelPosition(length, length, length)
 
     @property
-    def cone_resolution(self) -> int:  # numpydoc ignore=RT01
-        """Return or set the resolution of the cone tip.
+    def tip_resolution(self) -> int:  # numpydoc ignore=RT01
+        """Resolution of the axes tips.
 
         Examples
         --------
         >>> import pyvista as pv
-        >>> axes = pv.Axes()
-        >>> axes.axes_actor.cone_resolution
+        >>> axes_actor = pv.AxesActor()
+        >>> axes_actor.tip_resolution
         16
-        >>> axes.axes_actor.cone_resolution = 24
-        >>> axes.axes_actor.cone_resolution
+        >>> axes_actor.tip_resolution = 24
+        >>> axes_actor.tip_resolution
         24
 
         """
-        return self.GetConeResolution()
+        # Get cone value and assume value is the same for sphere
+        resolution = self.GetConeResolution()
+
+        # Make sure our assumption is true and reset value for cone and sphere
+        self.tip_resolution = resolution
+
+        return resolution
+
+    @tip_resolution.setter
+    def tip_resolution(self, resolution: int):  # numpydoc ignore=GL08
+        self.SetConeResolution(resolution)
+        self.SetSphereResolution(resolution)
+
+    @property
+    def cone_resolution(self) -> int:  # numpydoc ignore=RT01
+        """Resolution of the axes cone tips.
+
+        .. deprecated:: 0.43.0
+            This parameter is deprecated and will be completely removed
+            in version 0.46.0.
+
+            Use `tip_resolution` instead.
+        """
+        warnings.warn(
+            "Use of `cone_resolution` is deprecated and will be removed in 0.46.0. "
+            "Use `tip_resolution` instead.",
+            PyVistaDeprecationWarning,
+        )
+        return self.tip_resolution
 
     @cone_resolution.setter
-    def cone_resolution(self, res: int):  # numpydoc ignore=GL08
-        self.SetConeResolution(res)
+    def cone_resolution(self, resolution: int):  # numpydoc ignore=GL08
+        warnings.warn(
+            "Use of `cone_resolution` is deprecated and will be removed in 0.46.0. "
+            "Use `tip_resolution` instead.",
+            PyVistaDeprecationWarning,
+        )
```

Comment:
Please make sure you add the comment to help us grep for deprecations during releases.

Ref https://github.com/pyvista/pyvista/blob/main/CONTRIBUTING.rst#deprecating-features-or-other-backwards-breaking-changes